var/spack/repos/builtin/packages/perl-termreadkey/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PerlTermreadkey(PerlPackage):
```

Comment:
I see an existing `perl-term-readkey` package - would that be suitable in place of this?

Reply:
That looks like the same package, with this PR having a newer version, but again, why the extra '-'? Should I merge the new version into the existing package, or rename the existing one to not use the extra '-'?

Reply:
I will merge the update into the existing file.

Reply:
This has been committed.